docs/user-guide/advanced/model_registry.md

```diff
@@ -0,0 +1,80 @@
+# Backbone Model Registry
```

Comment:
also add a link to this page to `docs/user-guide/index.md`

Reply:
done